testing/font-siji/APKBUILD

```diff
@@ -0,0 +1,29 @@
+# Contributor: Taner Tas <taner76@gmail.com>
+# Maintainer: Taner Tas <taner76@gmail.com>
+pkgname=font-siji
+pkgver=20171022_git
+_pkgver="9d88311bb127b21672b2d4b43eed1ab0e494f143"
+pkgrel=0
+pkgdesc="Siji is an iconic bitmap font based on the Stlarch font with additional glyphs."
+url="https://github.com/stark/siji/"
+arch="noarch"
+license="GPL-2.0-only"
+depends="encodings font-alias mkfontscale mkfontdir fontconfig"
+#subpackages="$pkgname-dev $pkgname-doc"
+options="!check" # No test suite
+source="
+	$pkgname-$pkgver.tar.gz::https://github.com/stark/siji/archive/$_pkgver.tar.gz
+	"
+builddir="$srcdir/${pkgname/font-/}-$_pkgver"
+
+package() {
+	cd "$builddir"
+	install -Dm644 pcf/siji.pcf "$pkgdir"/usr/share/fonts/misc/siji.pcf
+#	install -Dm755 view.sh "$pkgdir"/usr/share/font-siji/view.sh
+}
+
+_package() {
+	cd "$builddir"
+}
```

Comment:
please remove `_package()` function

Reply:
upss. so many mistakes for such small abuild.